bionlp/evaluate/evaluation.py

```diff
@@ -75,7 +76,9 @@ def get_confusion_matrix(true, predicted, is_final_eval=False, final_eval_out_fi
 
 
 def pickle_confusion_matrix(confusion_matrix, path):
-    with open(path, "wb") as cm_f:
+    # TODO: Adding the current timestamp is only a temporary change
+    path_with_time = path + "__" + strftime("%Y-%m-%d_%H:%M:%S", gmtime())
```

Comment:
this creates two difficulties:
 1. The `:` in the filename is converted to a `/`, which is annoying when loading the file
 2. The timestamp is appended after the extension `.pkl`, which also is not optimal.
I would suggest to use
```
name, ext = os.path.splitext(path)
path_with_time = '{0}_{1}{2}'.format(name, int(time.time()), ext)
```